flask/entrypoint.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/sh
 #
 # run flask-migrate scripts and start gunicorn
```

Comment:
please change this comment, we are no longer running flask-migrate, which is from SQL Alchemy, since we are using mongodb now